test/simplifyExpression/multiplyFractionsSearch/multiplyFractionsSearch.test.js

```diff
@@ -10,7 +10,9 @@ describe('multiplyFractions', function () {
   const tests = [
     ['3 * 1/5 * 5/9', '(3 * 1 * 5) / (5 * 9)'],
     ['3/7 * 10/11', '(3 * 10) / (7 * 11)'],
-    ['2 * 5/x', '(2 * 5) / x']
+    ['2 * 5/x', '(2 * 5) / x'],
+    ['2 * (5/x)', '(2 * 5) / x'],
+    ['(5/x) * (2/x)', '(5 * 2) / (x * x)'],
```

Comment:
Yup good call, added (except for the case I don't support yet, re: comment above)